### ISSUE TITLE
TextView issue after human raises the bet

### ISSUE THREAD

**Issue by dendinge21** [bellosil21/pokerTexasHoldEm#4]
Step to recreate
1. Run Game
2. Wait till human player is the big blind
3. raise the bet to 40 
4. the AI's "Last Move" text view shows that they checked when the total pot when up and their money went down. By rules they should call or raise the bet. Could make the human confused.

**Comment by marcial21** [comment]
We will look into this.

**Comment by hoser21** [comment]
This functionality is as desired. I believe what happened was the AI's last move was to call your bet, but the AI was the last player to move in the phase. As a result, the next phase started, and this AI may had gone first by immediately checking. Hence, the human player did not have enough time to read that the AI called in the previous round.

We fixed this by making the AI's sleep for a small amount after sending an action, so the human player can read their action. Yet, this is a small time that we made the AI sleep, so if the human player got distracted, they may miss this notification. We do not want to make the AI sleep longer since we believe that it slows down the game by too much.
